File: js/16171.js
const fs = require('fs');

const inputs = fs.readFileSync('example.txt').toString().trim().split('\n');

const s = inputs[0]
  .split('')
  .filter((e) => !('0' <= e && e <= '9'))
  .join('');

const target = inputs[1];

console.log(+s.includes(target));

File: c++/16171.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
  freopen("example.txt", "r", stdin);
  string s, target;

  cin >> s >> target;

  string newS = "";

  for (int i = 0; i < s.length(); i++) {
    if ('0' <= s[i] && s[i] <= '9') {
      continue;
    }
    newS += s[i];
  }

  int ans = 0;

  for (int i = 0; i < newS.length(); i++) {
    if (newS[i] == target[0]) {
      bool flag = false;
      for (int j = i + 1; j < i + target.length(); j++) {
        if (newS[j] != target[j - i]) {
          flag = true;
          break;
        }
      }
      if (!flag) {
        ans = 1;
        break;
      }
    }
  }

  cout << ans;

  return 0;
}